Gitweb:     http://git.kernel.org/linus/9f800de38b05d84809e89f16671d636a140eede7
Commit:     9f800de38b05d84809e89f16671d636a140eede7
Parent:     123bf0e2eddcda36a33bdfc87aa1fb07229f07b5
Author:     Joerg Roedel <joerg.roedel@amd.com>
AuthorDate: Mon Nov 23 12:45:25 2009 +0100
Committer:  Joerg Roedel <joerg.roedel@amd.com>
CommitDate: Mon Nov 23 12:45:25 2009 +0100

    x86/amd-iommu: un__init iommu_setup_msi
    
    This function may be called on the resume path and can not
    be dropped after booting.
    
    Cc: stable@kernel.org
    Signed-off-by: Joerg Roedel <joerg.roedel@amd.com>
---
 arch/x86/kernel/amd_iommu_init.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/x86/kernel/amd_iommu_init.c b/arch/x86/kernel/amd_iommu_init.c
index 0d4581e..72bdbda 100644
--- a/arch/x86/kernel/amd_iommu_init.c
+++ b/arch/x86/kernel/amd_iommu_init.c
@@ -926,7 +926,7 @@ static int __init init_iommu_all(struct acpi_table_header *table)
  *
  ****************************************************************************/
 
-static int __init iommu_setup_msi(struct amd_iommu *iommu)
+static int iommu_setup_msi(struct amd_iommu *iommu)
 {
 	int r;
 
